src/utils/QCCalculator.cpp

```diff
@@ -396,29 +391,48 @@ class TOPPQCCalculator :
     at.colTypes.push_back("MS:1000285");
     UInt max = 0;
     Size below_10k = 0;
-    for (Size i = 0; i < exp.size(); ++i)
-    {
-      if (exp[i].getMSLevel() == 1)
+    std::vector<OpenMS::Chromatogram> chroms = exp.getChromatograms();
+    if (chroms.size() == 1) //real TIC from the mzML
```

Comment:
did you check if there is the TIC CV stored in the chromatogram? I am not 100% but I think I recall something like this